db/seeds.rb

```diff
@@ -1,5 +1,22 @@
 # frozen_string_literal: true
-# This file should contain all the record creation needed to seed the database
-# with its default values.
-# The data can then be loaded with the rails db:seed command
-# (or created alongside the database with db:setup).
+# rubocop:disable Rails/Output
+
+puts 'Generating seed data....'
+
+Generator.generate_admin(email: 'email@email.com', password: 'passw0rd')
+Generator.generate(model: 'building', count: 2)
+Generator.generate(model: 'suite', count: 15)
+
+# generate bedrooms
+Generator.generate(model: 'room', count: 30)
+# generate common rooms
+Generator.generate(model: 'room', count: 10, beds: 0)
+
+Generator.generate(model: 'user', count: 15)
+
+# fix this eventually so that we never generate empty suites
+Suite.where(size: 0).destroy_all
```

Comment:
I feel like the only way to avoid this is by somehow generating suites with rooms in the `SuiteGenerator` itself. I'm kind of ok with that, tbh.